tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -563,3 +563,28 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
     );
   });
 }
+
+describe('Test GitRepo Bundle do not show hash mismatch error.', { tags: '@p1'}, () => {
+
+  qase(195,
+
+    it("Fleet-195: Test GitRepo bundle hash is not mismatch.", { tags: '@fleet-195' }, () => {
+
+      const repoName = "test-bundle-hash-mistmatch"
+      const path = "qa-test-apps/bundle-hash-test"
+
+      cy.addFleetGitRepo({ repoName, repoUrl, branch, path });
+      cy.clickButton('Create');
+      cy.verifyTableRow(0, 'Active', repoName);
+      cy.checkGitRepoStatus(repoName, '1 / 1', '1 / 1');
+
+      // Verify that bundle is showing Active state and not showing hash mismatch error.
+      cy.continuousDeliveryBundlesMenu();
+      cy.filterInSearchBox(repoName);
+      cy.verifyTableRow(0, 'Active', repoName);
+
+      cy.deleteAllFleetRepos();
```

Comment:
Can we omit this, given that by default we have a deletion mechanism at the beginning of each test?

Reply:
Okay, you're right, I will remove it.